fixtures/webversions.py

```diff
@@ -0,0 +1,48 @@
+from dotenv import load_dotenv
+import os
+import pytest
+import urllib.request
+from ast import literal_eval
+load_dotenv(verbose = True)
+url_addr_version = os.environ['URL_ADDR_VERSION']
```

Comment:
You are using `URL_ADDR VERSION` here... environment variables should be handled somewhere else. What could you do differently?